### ISSUE TITLE
Heroic on Flatpak complains that Wine isn't configured

### ISSUE THREAD

**Issue by guihkx** [Heroic-Games-Launcher/HeroicGamesLauncher#1321]
### Describe the bug

I'm trying to launch Rocket League on the Flatpak build of Heroic, but I get these two errors instead:

- *Error when launching the game, check the logs!*
- *No Wine Version Selected. Check Game Settings!*

### Add logs

Apparently, the logs in *Settings* > *Log* doesn't include `stderr` output, which leaves out a lot of useful information.

Instead, I'll post the terminal output when I run `flatpak run com.heroicgameslauncher.hgl`, which includes everything:

```
Gtk-Message: 00:12:00.511: Failed to load module "canberra-gtk-module"
Gtk-Message: 00:12:00.512: Failed to load module "canberra-gtk-module"
[3:0516/001200.528427:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:0516/001200.528473:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
(00:12:00) DEBUG:   [Legendary]:       Running Legendary command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
[42:0516/001200.631078:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
hostname: invalid option -- 'f'
Try 'hostname --help' for more information.
(00:12:01) INFO:    [Legendary]:       Legendary location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(00:12:01) INFO:    [Gog]:             GOGDL location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(00:12:01) INFO:    [Backend]:         
  Heroic Version: 2.3.1 Brook
  Legendary Version:  0.20.25 Our Benefactors
  OS: Freedesktop.org KERNEL: 5.17.7-arch1-1 ARCH: x64
  CPU: Intel Core™ i5-3570K @3.4 GOVERNOR: powersave
  RAM: Total: 7.68 GiB Available: 4.66 GiB
  GRAPHICS: GPU0: GK106 [GeForce GTX 660]  VRAM: 32MB DRIVER:  
  PROTOCOL: x11
  
APPIMAGE env is not defined, current application is not an AppImage
LaunchProcess: failed to execvp:
xdg-settings
(00:12:01) ERROR:   [Backend]:         Failed to register protocol with OS.
[3:0516/001201.267265:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[3:0516/001201.267389:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[3:0516/001201.267518:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[3:0516/001201.431259:ERROR:browser_main_loop.cc(269)] Gdk: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
(00:12:02) INFO:    [Frontend]:        Refreshing Library
(00:12:02) INFO:    [Legendary]:       Refreshing library...
(00:12:02) INFO:    [Legendary]:       Refreshing Epic Games...
(00:12:02) INFO:    [Legendary]:       Refreshing Epic Games...
(00:12:03) DEBUG:   [Legendary]:       Running Legendary command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list
(node:131) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 setGameStatus listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit
(Use `exe --trace-warnings ...` to show where the warning was created)
(00:12:05) INFO:    [Legendary]:       Updating game list
(00:12:05) INFO:    [Legendary]:       Game List Updated
(00:12:05) INFO:    [Legendary]:       Checking for game updates: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(00:12:05) DEBUG:   [Legendary]:       Running Legendary command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(00:12:06) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(00:12:06) DEBUG:   [Legendary]:       Running Legendary command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
hostname: invalid option -- 'f'
Try 'hostname --help' for more information.
(00:12:06) INFO:    [Backend]:         Log was written to /home/gui/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/Sugar-lastPlay.log
(00:12:06) INFO:    [Frontend]:        Refreshing Library
(00:12:06) INFO:    [Legendary]:       Refreshing library...
(00:12:06) INFO:    [Legendary]:       Updating game list
(00:12:06) INFO:    [Legendary]:       Game List Updated
(00:12:07) INFO:    [Legendary]:       Found 0 game(s) to update
(00:12:07) INFO:    [Gog]:             Found 0 game(s) to update


(node:131) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 setGameStatus listeners added to [EventEmitter]. Use emitter.setMaxListeners() to increase limit
(00:16:22) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(00:16:22) DEBUG:   [Legendary]:       Running Legendary command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
hostname: invalid option -- 'f'
Try 'hostname --help' for more information.
(00:16:23) INFO:    [Backend]:         Log was written to /home/gui/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/Sugar-lastPlay.log
(00:16:23) INFO:    [Frontend]:        Refreshing Library
(00:16:23) INFO:    [Legendary]:       Refreshing library...
(00:16:23) INFO:    [Legendary]:       Updating game list
(00:16:23) INFO:    [Legendary]:       Game List Updated
```

### Steps to reproduce

1. Install Rocket League on the Flatpak build of Heroic
2. Install the latest Wine version available using Heroic's Wine Manager
3. Try launching the game

### Expected behavior

No error.

### Screenshots

- The two errors after trying to launch Rocket League:
    ![image](https://user-images.githubusercontent.com/626206/168514470-a2bf2aeb-4b6b-4fb1-b73b-8f9bb4e17957.png)
    ![image](https://user-images.githubusercontent.com/626206/168514107-432d216f-ba88-4876-8509-a60ceba155a3.png)

- My Rocket League settings:
    ![image](https://user-images.githubusercontent.com/626206/168514536-b1b55216-3838-4ce4-b1e8-43b4e8b9f3d2.png)

- Wine Manager:

    ![image](https://user-images.githubusercontent.com/626206/168514826-785bde78-05c1-4e7d-8185-8d76f8ceabee.png)

- Flatpak overrides:

    ```
    $ cat ~/.local/share/flatpak/overrides/com.heroicgameslauncher.hgl 
    [Context]
    filesystems=/mnt/HDDLinux/Games/heroic
    ```

### System Information

- OS:  Arch Linux
- Heroic Version: v2.3.1 (Flatpak)

**Comment by flavioislima** [comment]
Can you add the game logs as well?

**Comment by guihkx** [comment]
Yep, that did it! I picked the `lutris-fshack-7.2` build from the drop-down, and then immediately switched back to the `GE-Proton7-11` one. Then I tried opening `winecfg` and the prefix was created successfully. The game launched after that as well!

What I don't understand is how this `fshack` build is even listed there, I don't remember installing it... 🤔 

![image](https://user-images.githubusercontent.com/626206/168636424-49b04a04-7c28-4677-b5b5-4d7682d6be35.png)
